### ISSUE TITLE
Messages sent but chat page doesn't reload with them. 

### ISSUE THREAD

**Issue by YeisonHunt** [nahid/talk-example#40]
1: When I clicked Send button. 

I don't use docker for this example, could be that the reason?...
Actually using XAMPP, running the migrations on port 8088 with php artisan serve.

Psdata: 

The messeges are stored in db but not retrieved automatically.
![1](https://user-images.githubusercontent.com/20606919/43413066-95a3b30a-93f4-11e8-8761-ac233f2cfe1f.PNG)

**Comment by nahid** [comment]
yes thats the reason. for realtime messaging talk use redis to queuing message.
